app/modules/individuals/parameters.py

```diff
@@ -26,29 +29,67 @@ class Meta(schemas.DetailedIndividualSchema.Meta):
 
 class PatchIndividualDetailsParameters(PatchJSONParameters):
     # pylint: disable=abstract-method,missing-docstring
-    OPERATION_CHOICES = (PatchJSONParameters.OP_REPLACE,)
+    OPERATION_CHOICES = (
+        PatchJSONParameters.OP_REPLACE,
+        PatchJSONParameters.OP_ADD,
+        PatchJSONParameters.OP_REMOVE,
+    )
+
+    # Is this weird? I need to modify EDM and Houston records
+    PATH_CHOICES_HOUSTON = ['encounters']
+
+    PATH_CHOICES_EDM = ['encounters']
+
+    PATH_CHOICES = tuple(
+        '/%s' % field for field in (PATH_CHOICES_EDM + PATH_CHOICES_HOUSTON)
+    )
+
+    @classmethod
+    def remove(cls, obj, field, value, state):
+        has_permission = rules.ObjectActionRule(obj, AccessOperation.WRITE).check()
+        ret_val = False
+        if has_permission:
+            if field == 'encounters':
+                for encounter_guid in value:
+                    if is_valid_uuid(encounter_guid):
+                        from app.modules.encounters.models import Encounter
+
+                        encounter = Encounter.query.filter(
```

Comment:
why don't you use Encounter.query.get(encounter_guid). 
There is no need for the filter/first or even the is_valid_uuid check. If it's not a valid uuid, it's not going to be got.

Reply:
I'm still brewing on this one. Very cautious about too many database hits, although this is a basic one.

Reply:
My point still stands about the database hit, but I've changed this for now. If it is valid, we need to get the encounter anyway here. And I sure hope we aren't sending that many invalid guids here.